compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BTupleMemberType.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wso2.ballerinalang.compiler.semantics.model.types;
+
+import org.wso2.ballerinalang.compiler.semantics.model.symbols.BVarSymbol;
+
+/**
+ * {@code {@link BTupleMemberType }} represents a member in a tuple.
+ *
+ */
+public class BTupleMemberType {
```

Comment:
I still feel like suffixing this with `Type` is a bit misleading. Since it is not just the type. Shall we say `BTupleMember`?

Reply:
Basically the parallel to this would be `BField`.